app/services/search/postgres/reading_list.rb

```diff
@@ -0,0 +1,121 @@
+module Search
+  module Postgres
+    class ReadingList
+      ATTRIBUTES = [
+        "articles.cached_tag_list",
+        "articles.crossposted_at",
+        "articles.path",
+        "articles.published_at",
+        "articles.reading_time",
+        "articles.title",
+        "articles.user_id",
+        "reactions.id AS reaction_id",
+        "reactions.user_id AS reaction_user_id",
+      ].freeze
+      USER_ATTRIBUTES = %i[id name profile_image username].freeze
+
+      DEFAULT_PER_PAGE = 60
+      DEFAULT_STATUSES = %w[confirmed valid].freeze
+
+      def self.search_documents(user, term: nil, statuses: [], tags: [], page: 0, per_page: DEFAULT_PER_PAGE)
+        return {} unless user
+
+        statuses = statuses.presence || DEFAULT_STATUSES
+        tags = tags.presence || []
+
+        # NOTE: [@rhymes] we should eventually update the frontend
+        # to start from page 1
+        page = page.to_i + 1
+        per_page = [(per_page || DEFAULT_PER_PAGE).to_i, 100].min
```

Comment:
I'm assuming we chose `100` intentionally here -- should we extract that into a const as well (or at least leave a comment as to why we chose it)?

Reply:
Done